instance.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_instance" "web" {
+  ami           = "ami-0022f774911c1d690"
+  instance_type = "t2.micro"
+  #security_groups = ["aws_security_group.tf_sg.id"]
+  key_name = "training"
+  tags = {
+    Name = "HelloWorld"
+  }
+  /*user_data = <<EOF
+#! /bin/bash
+yum update -y
+yum install -y httpd
+systemctl start httpd && systemctl enable httpd
+echo “Hello World from $(hostname -f)” > /var/www/html/index.html
+EOF*/
+
+  user_data = file("script.sh")
+
+}
+
+resource "aws_security_group" "tf_sg" {
+  name        = "Security group using terraform"
+  description = "Security group using terraform"
+  vpc_id      = "vpc-0b98a050f32fec47f"
```

Comment:
Run `terraform destroy` and `terraform apply` and you will see why this will not work